packages/terra-list/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 * Added
+  * Added cyclic navigation to support keyboard usage with arrow keys.
```

Comment:
```suggestion
  * Added arrow key navigation support for list items.
```
Please add space between Unreleased and Added section

Reply:
Changes done. Thanks